CMakeLists.txt

```
@@ -18,7 +18,7 @@ set(APPLE_SUPPRESS_X11_WARNING ON)
 
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
-find_package(QT 6.5 NAMES Qt6 COMPONENTS Core REQUIRED)
+find_package(QT 6.7 NAMES Qt6 COMPONENTS Core REQUIRED)
```

if we do this in PR https://github.com/owncloud/client/pull/11994. Then we might get the CI running for this PR
```suggestion
find_package(QT 6.5 NAMES Qt6 COMPONENTS Core REQUIRED)
```